lib/lure/git.go

```diff
@@ -84,3 +85,24 @@ func (gitRepo GitRepo) LogCommitsBetween(baseRev string, secondRev string) ([]st
 	lines := strings.Split(out, "\n")
 	return append(lines[:0], lines[:len(lines)-1]...), nil
 }
+
+func (gitRepo GitRepo) GetActiveBranches() ([]string, error) {
+	out, err := gitRepo.Cmd("branch", "-r")
+	if err != nil {
+		return nil, err
+	}
+	// remove first line which is always origin/HEAD -> origin/master and last empty line (trim space)
```

Comment:
I'm not sure this is true, tested this in a local repo and got different results. it may be better to explicitly filter it out

Reply:
was your repo strictly local or was it a local of a remote one from github or bitbucket?

Reply:
I can repro on a freshly cloned repo from bitbucket, when the branch name starts with a number.

Reply:
I'll fix that